### PR TITLE
components colors & refactor

### PR DESCRIPTION
**Pull request by JulianWielga**

### CODE REVIEW

File: ui/submodules/packages/components/src/settings/rootProviders.tsx [hunk 1]
```diff
@@ -4,14 +4,42 @@ import { QueryClientProvider } from "react-query";
...
+    return (element) => {
+        if (element.type !== "rule") {
+            return;
+        }
+
+        if (element.root?.type === "@keyframes") {
+            return;
+        }
+
+        if (!element.parent || (element.props.length === 1 && element.value.charCodeAt(0) !== 58) || !element.length) {
```
Comment (arkadius) [inline ui/submodules/packages/components/src/settings/rootProviders.tsx:23]: please add comment

**Review by arkadius — APPROVED** [review]

File: ui/submodules/packages/components/src/components/view.tsx [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
+                <Stack direction="column" justifyContent="center" height="100%" spacing={2} overflow="hidden" p={2}>
+                    <Routes>
+                        <Route path="/" element={<ListWithFilters />} />
+                        <Route
+                            path="/:componentId"
+                            element={
+                                <Typography variant="h2">
+                                    {`not yet... `}
+                                    <Link component={RouterLink} to={`/`} replace color="secondary">
+                                        go back!
```
Comment (lciolecki) [inline ui/submodules/packages/components/src/components/view.tsx:19]: Missing translation.

**Review by lciolecki — APPROVED** [review]
LGTM, but please add missing translation option.

File: ui/submodules/packages/components/src/components/view.tsx (hunk 2 above)
Comment (JulianWielga) [reply ui/submodules/packages/components/src/components/view.tsx:19]: it's temporary, not worth attention

**Pull request merged**